src/components/InfoPanel/InfoPanel.js

```diff
@@ -24,6 +24,8 @@ const expandedTopologyDimensions = () => {
 
 const infoPanelDimensions = () => {
   const vw = getViewportWidth();
+  // If changes are made to the 1580px breakpoint below then be sure to update
+  // the same breakpoint in _info-panel.scss.
```

Comment:
For future but I would connect this with the css via a classname

Reply:
These are the values used to pass to the topology dimensions so a class won't work as we actually need the numbers.